helper_scripts/cloudstack/build_run_deploy_test.sh

```diff
@@ -417,9 +421,18 @@ date
 # Run the tests
 if [ ${run_tests} -eq 1 ]; then
   echo "Running Marvin tests.."
-  bash -x /data/shared/helper_scripts/cloudstack/run_marvin_router_tests.sh ${marvinCfg}
+  if [ -z ${test_file} ]; then
+    # for backwards compatibility, default to 'run_marvin_router_tests.sh' if test file is not specified
+    test_file="run_marvin_router_tests.sh"
+  fi
+  if [ -f "/data/shared/helper_scripts/cloudstack/${test_file}" ]; then
```

Comment:
I could change it so it tries to treat the input as an absolute path and if the final does not exist, then it checks if it is in this folder like it currently functions.